app-office/lyx/lyx-2.3.4.2.ebuild

```diff
@@ -0,0 +1,174 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+MY_P="${P/_}"
+inherit desktop flag-o-matic font python-single-r1 qmake-utils toolchain-funcs xdg-utils
+
+DESCRIPTION="WYSIWYM frontend for LaTeX, DocBook, etc."
+HOMEPAGE="https://www.lyx.org/"
+SRC_URI="ftp://ftp.lyx.org/pub/lyx/stable/2.3.x/${MY_P}.tar.xz
+	ftp://ftp.lyx.org/pub/lyx/devel/lyx-2.3/${MY_P}/${MY_P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86 ~x64-macos ~x86-macos"
+IUSE="aspell cups debug docbook dia dot enchant gnumeric html +hunspell +latex monolithic-build nls rcs rtf subversion svg l10n_he"
```

Comment:
It seems this ebuild has some long standing problems:
- `dia` USE flag is not used except in RDEPEND
- `dot` USE flag is not used except in RDEPEND
- `cups` USE flag is not used except in RDEPEND
- `gnumeric` USE flag is not used except in RDEPEND
- `html` USE flag is not used except in RDEPEND
- `rcs` USE flag is not used except in RDEPEND, but this one I found as optional dep at least in other distro as well.
- `rtf` USE flag is not used except in RDEPEND
- `subversion` USE flag is not used except in RDEPEND
- `svg` USE flag is not used except in RDEPEND

Reply:
Would it make sense to remove thes useflags?

Reply:
Leaving for future PRs